conf/ocsci/aws_upi_rhel_workers.yaml

```diff
@@ -0,0 +1,14 @@
+---
+ENV_DATA:
+  deployment_type: 'upi'
+  rhel_workers: true
+  rhcos_ami: 'ami-06c85f9d106577272'
+  rhel_worker_ami: 'ami-073251e3a0e36cebb' # RHEL7.7 as of now
+  # This repo is just for reference
+  OCP_INTERNAL_REPO_LATEST: 'https://mirror.openshift.com/enterprise/all/4.2/latest/RH7-RHAOS-4.2/x86_64/os/'
```

Comment:
I discussed this on the PR from Vijay #1037 , he has approach of using repo file and I commented that we cannot just stick with 4.2 version. So I am for @vavuthu approach, but have this repo file versioned for specific ocp version. So we can then decide based on installed ocp version which one repo file we will use.

Yep as there is work going on in parallel, we can go with this approach but open the issue and we need to consolidate this once both PRs are merged. So maybe you can open the issue for consolidating both PRs work and use common functionality and start updating in that issue what we need to do. So you can start with this point.  Valid for all below as well as I see 4.2 mentioned there a lot

Reply:
Oh sorry, it is a stale entry I forgot to delete. I am following as per Vijay's Idea of repofile. Cleaned the stale entries.